exporter/elasticsearchexporter/integrationtests/collector.go

```diff
@@ -0,0 +1,209 @@
+package integrationtests
```

Comment:
I'm not sure if this header is strictly needed but I've seen it on other files, including test files.

```suggestion
// Copyright The OpenTelemetry Authors
// SPDX-License-Identifier: Apache-2.0

package integrationtests
```

Reply:
Thanks for catching this. I have pushed a fix to add license headers to all files.